src/core_plugins/metrics/public/components/panel_config/table.js

```diff
@@ -0,0 +1,150 @@
+  }
+
+  render() {
+    const { selectedTab } = this.state;
+    const defaults = { drilldown_url: '', filter: '', pivot_label: '', pivot_rows: 10 };
+    const model = { ...defaults, ...this.props.model };
+    const handleSelectChange = createSelectHandler(this.props.onChange);
+    const handleTextChange = createTextHandler(this.props.onChange);
+    let view;
+    if (selectedTab === 'data') {
+      view = (
+        <div>
+          <div className="vis_editor__table-pivot-fields">
+            <div className="vis_editor__container">
+              <div className="vis_ediotr__vis_config-row">
+                <p>
+                  For the table visualization you need to define a field to
+                  group by using a terms aggregation.
+                </p>
+              </div>
+              <div className="vis_editor__vis_config-row">
+                <div className="vis_editor__label">Group By Field</div>
+                <div className="vis_editor__row_item">
+                  <FieldSelect
+                    fields={this.props.fields}
+                    value={model.pivot_id}
+                    indexPattern={model.index_pattern}
+                    onChange={handleSelectChange('pivot_id')}
+                  />
+                </div>
+                <div className="vis_editor__label">Column Label</div>
+                <input
+                  className="vis_editor__input-grows"
+                  type="text"
+                  onChange={handleTextChange('pivot_label')}
+                  value={model.pivot_label}
+                />
+                <div className="vis_editor__label">Rows</div>
+                <input
+                  className="vis_editor__input-number"
+                  type="number"
+                  onChange={handleTextChange('pivot_rows')}
+                  value={model.pivot_rows}
+                />
+              </div>
+            </div>
+          </div>
+          <SeriesEditor
+            fields={this.props.fields}
+            model={this.props.model}
+            name={this.props.name}
+            onChange={this.props.onChange}
+          />
+        </div>
+      );
+    } else {
+      view = (
+        <div className="vis_editor__container">
+          <div className="vis_editor__vis_config-row">
+            <div className="vis_editor__label">Item Url (This supports mustache templating.
+              <code>{'{{key}}'}</code> is set to the term)
+            </div>
+            <input
+              className="vis_editor__input-grows"
+              onChange={handleTextChange('drilldown_url')}
+              value={model.drilldown_url}
+            />
+          </div>
+          <IndexPattern
+            fields={this.props.fields}
+            model={this.props.model}
+            onChange={this.props.onChange}
+          />
+          <div className="vis_editor__vis_config-row">
+            <div className="vis_editor__label">Panel Filter</div>
```

Comment:
This should most likely be a `<label>` with a `for` attribute pointing to the `input`.